rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestControllerHelper.java

```diff
@@ -684,6 +694,24 @@ public License convertToEmbeddedLicense(String licenseId) {
         return embeddedLicense;
     }
 
+    public Release convertToEmbeddedLinkedRelease(Release release) {
+        Release embeddedRelease = new Release();
+        embeddedRelease.setId(release.getId());
+        embeddedRelease.setMainlineState(release.getMainlineState());
+        embeddedRelease.setClearingState(release.getClearingState());
+        embeddedRelease.setMainLicenseIds(release.getMainLicenseIds());
+        embeddedRelease.setOtherLicenseIds(release.getOtherLicenseIds());
+        embeddedRelease.setName(release.getName());
+        embeddedRelease.setComponentType(release.getComponentType());
+        embeddedRelease.setVersion(release.getVersion());
+        embeddedRelease.setType(null);
+        return embeddedRelease;
+    }
+
+    public void addEmbeddedProjectReleases(HalResource halResource, List<EntityModel<Release>> releases) {
+        halResource.addEmbeddedResource("sw360:release", releases);
```

Comment:
Keep `sw360:release` if you can

Reply:
It's already sw360:release.

Reply:
@rudra-superrr Oh, sorry I mean `sw360:releases`

Reply:
If I keep sw360:releases, then nested list will come in response structure.

![image](https://github.com/eclipse-sw360/sw360/assets/58290634/93d92175-7d5f-4bfd-a8df-98e8d6e90a64)

Reply:
> If I keep sw360:releases, then nested list will come in response structure.
> 
> ![image](https://user-images.githubusercontent.com/58290634/257724085-93d92175-7d5f-4bfd-a8df-98e8d6e90a64.png)

@rudra-superrr, I got it, thanks